Peekabook/Peekabook/Network/DTO/NaverSearch/NaverSearchResponse.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  NaverSearchResponse.swift
+//  Peekabook
+//
+//  Created by 고두영 on 2023/01/10.
+//
+
+//import Foundation
+//
+//// 네이버 책검색 API 호출 메소드
+//class DataManager {
+//    static let shared: DataManager = DataManager()
+//    var searchResult: NaverSearchResponse?
+//
+//    private init() {
+//
+//    }
+//}
+//
+//// MARK: - NaverSearchResponse
+//
+//struct NaverSearchResponse: Codable {
+//    let items: [BookDetail]
+//}
+//
+//// MARK: - Item
+//
+//struct BookDetail: Codable {
+//    let title: String
+//    let image: String
+//    let author: String
+//}
+
+import Foundation
+
+class DataManager {
+    static let shared : DataManager = DataManager()
+    var searchResult : PostBook?
+    
+    private init() {
+        
+    }
```

Comment:
p3. 
안에 코드 없으면 이렇게 한 줄로 적어주세용

```suggestion
   private init() { }
```

Reply:
question.
근데 이 코드는 왜 있을까여..?